pleskdistup/common/src/files.py

```diff
@@ -59,25 +60,31 @@ def get_last_lines(filename: PathType, n: int) -> typing.List[str]:
         return f.readlines()[-n:]
 
 
-def backup_file(filename: str) -> None:
+def backup_file(filename: str, ext: str = DEFAULT_BACKUP_EXTENSION) -> None:
     if os.path.exists(filename):
-        shutil.copy(filename, filename + ".bak")
+        shutil.copy(filename, filename + ext)
 
 
-def backup_exists(filename: str) -> bool:
-    return os.path.exists(filename + ".bak")
+def backup_exists(filename: str, ext: str = DEFAULT_BACKUP_EXTENSION) -> bool:
+    return os.path.exists(filename + ext)
 
 
-def restore_file_from_backup(filename: str, remove_if_no_backup: bool = False) -> None:
-    if os.path.exists(filename + ".bak"):
-        shutil.move(filename + ".bak", filename)
+def restore_file_from_backup(filename: str, remove_if_no_backup: bool = False,
+                             ext: str = DEFAULT_BACKUP_EXTENSION) -> None:
```

Comment:
Too much indent.

Reply:
It was automatic but looks good IMO, I will try 2 shifts